src/main/java/pl/mimuw/zpp/quantumai/backendui/controller/dto/GraphDto.java

```diff
@@ -0,0 +1,11 @@
+package pl.mimuw.zpp.quantumai.backendui.controller.dto;
+
+import lombok.Builder;
+import pl.mimuw.zpp.quantumai.backendui.model.GraphType;
+
+@Builder
+public record GraphDto(
+        GraphType graphType,
+        Object graph
```

Comment:
Czemu nie używamy jakiejś wyższej klasy abstrakcji dla grafu zamiast `Object`?

Reply:
Nie mam lepszego pomysłu, alternatywnie myślałem o czymś typu
```
{
graphType: EUCLIDEAN,
euclideanGraph: { ... },
mapGraph: null
}
```
oraz
```
{
graphType: MAP,
euclideanGraph: null,
mapGraph: { ... }
}
```

Reply:
A nie można po prostu zrobić klasy abstrakcyjnej `Graph` zamiast `Object` po której będą dziedziczyły  nasze oba typy grafów?

Reply:
może być, tylko nie wiem, czy to coś zmieni z perspektywy jsona